### PR TITLE
[Forome-982] Actualize e2e tests 2

### PR DESCRIPTION
**Pull request by viraxslot**
- actualized tests
- fixed selectors
- API endpoints moved to constants
- small refactoring

### CODE REVIEW

File: src/pages/filter/common/view-variants/variant-content/variant-content.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { Icon } from '@ui/icon'
 import { Loader } from '@ui/loader'
 import { VariantAspectsLayoutGallery } from '@components/variant-aspects-layout'
 import { TAspectDescriptor } from '@service-providers/dataset-level'
+import { ReturnedVariantsDataCy } from '../../../../../components/data-testid/returned-variants'
```
Comment (Lapk1n) [inline src/pages/filter/common/view-variants/variant-content/variant-content.tsx]: could be shorter {...} from '@components/data-testid...'

File: src/components/units-list/units-list-group/units-list-group.tsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ import { TUnitGroup } from '@store/stat-units'
 import { DropdownArrow } from '@ui/dropdown-arrow'
 import { PredictionPowerIndicator } from '@components/prediction-power-indicator'
 import { TPropertyStatus } from '@service-providers/common'
+import { DecisionTreesResultsDataCy } from '../../data-testid/decision-tree-results.cy'
```
Comment (Lapk1n) [inline src/components/units-list/units-list-group/units-list-group.tsx]: the same {...} from '@components/data-testid...'

File: src/components/units-list/units-list-controls/units-list-controls.tsx [hunk 3]
```diff
@@ -6,6 +6,7 @@ import cn from 'classnames'
 import { t } from '@i18n'
 import { Icon } from '@ui/icon'
 import { InputSearch } from '@components/input-search'
+import { DecisionTreesResultsDataCy } from '../../data-testid/decision-tree-results.cy'
```
Comment (Lapk1n) [inline src/components/units-list/units-list-controls/units-list-controls.tsx]: the same {...} from '@components/data-testid...'

File: src/components/solution-control/solution-control-popover/solution-control-popover.tsx [hunk 4]
```diff
@@ -9,6 +9,7 @@ import { Popover } from '@ui/popover'
 import { IPopoverBaseProps } from '@ui/popover/popover.interface'
 import { popoverOffset } from '@pages/ws/ws.constants'
 import { ISolutionEntryDescription } from '@service-providers/common'
+import { DecisionTreesMenuDataCy } from '../../data-testid/decision-tree-menu.cy'
```
Comment (Lapk1n) [inline src/components/solution-control/solution-control-popover/solution-control-popover.tsx]: the same {...} from '@components/data-testid...'

File: src/components/solution-control/solution-control-button/solution-control-button.tsx [hunk 5]
```diff
@@ -6,6 +6,7 @@ import cn from 'classnames'
 import { t } from '@i18n'
 import { Icon } from '@ui/icon'
 import { Loader } from '@ui/loader'
+import { DecisionTreesMenuDataCy } from '../../data-testid/decision-tree-menu.cy'
```
Comment (Lapk1n) [inline src/components/solution-control/solution-control-button/solution-control-button.tsx]: the same {...} from '@components/data-testid...'

**Review by Lapk1n — APPROVED** [review]

**Review by AlMaQntr — COMMENTED** [review]
I think, we should move `data-testid` dir outside of `src` dir.
Next, create `index.ts` inside it, where exports all identifiers.
And next, create an alias to `data-testid` dir e.g. `@data-testid`
As a result, we will have all test id imports like this:
```
import { SomeId, SomeOtherId } from '@data-testid'
```

File: src/pages/filter/common/view-variants/variant-content/variant-content.tsx (hunk 1 above)
Comment (viraxslot) [reply src/pages/filter/common/view-variants/variant-content/variant-content.tsx]: Fixed using @AlMaQntr suggestion

**Comment by viraxslot** [comment]
> I think, we should move `data-testid` dir outside of `src` dir.

@AlMaQntr Craco doesn't allow to import something outside the `src` folder so I moved `data-testid` folder to `src` and added an alias and `index.ts` with all exports.

**Review by AndreyUstyumenko — APPROVED** [review]

**Review by Lapk1n — APPROVED** [review]

**Pull request merged**